src/Logikfabrik.Umbraco.Jet/Mappings/DefaultDataTypeDefinitionMapping.cs

```diff
@@ -118,39 +122,71 @@ private static Type GetNullableType(Type type)
         /// <summary>
         /// Gets the supported hints.
         /// </summary>
+        /// <param name="umbracoPropertyValueConvertersEnabled">The value of EnablePropertyValueConverters in umbracoSettings.config</param>
         /// <returns>The supported hints.</returns>
-        private static IDictionary<string, KeyValuePair<Type, DataTypeDefinition>> GetSupportedHints()
+        private static IDictionary<string, KeyValuePair<Type, DataTypeDefinition>> GetSupportedHints(bool umbracoPropertyValueConvertersEnabled)
```

Comment:
So this setting dictates whether to use the new pickers, or the old ones? Could you set up a small unit test for this?